File: c/1018.c
#include <stdio.h>

int main()
{
    int notes[] = {100, 50, 20, 10, 5, 2, 1};
    int n, times;
    scanf("%d", &n);

    printf("%d\n", n);
    for (int i = 0; i < 7; i++) {
        times = n / notes[i];
        n = n % notes[i];
        printf("%d nota(s) de R$ %d,00\n", times, notes[i]);
    }

	return 0;
}

File: lua/1018.lua
n = io.read("*n")
notes = {100, 50, 20, 10, 5, 2, 1}

print(n)
for i, v in pairs(notes) do 
    times = n // v;
    n = n % v;
    print(times .. " nota(s) de R$ " .. v .. ",00")
end
